.github/workflows/CD.yml

```
@@ -0,0 +1,36 @@
+name: CD
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: self-hosted
+    steps:
+      - name: SSH and Deploy
+        run: |
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)